RAG_supporters/dataset/torch_dataset.py

```diff
@@ -56,6 +79,14 @@ def __init__(
         return_embeddings: bool = True,
         chunksize: int = 10000,
         embedding_model_name: Optional[str] = None,
+        # Cluster steering parameters
+        steering_mode: Optional[Union[SteeringMode, str]] = None,
```

Comment:
Steering mode should be as optional list pickable modes, with probabilities of being used.